oai_agents/agents/rl.py

```diff
@@ -251,8 +249,7 @@ def wrap_agent(self, sb3_agent, name):
         return SB3Wrapper(sb3_agent, name, self.args)
 
     def get_experiment_name(self, exp_name):
-        all_train_teamtypes = [tag for tags_dict in self.teammates_collection.values() for tag in tags_dict.keys()]
-        return exp_name or 'train_' + '_'.join(all_train_teamtypes)
+        return exp_name or str(self.args.exp_dir) + '/' + self.name
```

Comment:
recommend putting parentheses around `or` statement to ensure it evaluates how intended
```
return (exp_name or str(self.args.exp_dir)) + '/' + self.name
```

Reply:
this was intentional because sometimes we may want to override the whole experiment_name :D

Reply:
ah whoops! That makes sense 😅